chain/vm/invoker.go

```diff
@@ -138,7 +138,7 @@ func (*Invoker) transform(instance Invokee) (nativeCode, error) {
 
 				inBytes := in[1].Interface().([]byte)
 				if err := DecodeParams(inBytes, param.Interface()); err != nil {
-					aerr := aerrors.Absorb(err, 1, "failed to decode parameters")
+					aerr := aerrors.Absorb(err, exitcode.SysErrSerialization, "failed to decode parameters")
```

Comment:
This should be `ErrSerialization`, `SysErrSerialization` doesn't exist any more.